dl-snapchat.html

```diff
@@ -0,0 +1,20 @@
+---
+background: dark
+layout: blank
+title: "Vibe Check"
+---
+<div class="page-vibe-check">
+  <div class="logo">
+      <img src="/assets/images/favicon/android-chrome-192x192.png"/>
+  </div>
+  <div class="body-text">
+    Turn&nbsp;your Snapchat stories&nbsp;into dreams.
+  </div>
+  <div class="button">
+    <a  href="https://thetrash.app.link/36xcKYfZr7" target="_blank">
+      <img class="off" src="/assets/images/appstorebutton_rainbow.svg"/>
+      <!-- <img class="on" src="/assets/images/appstorebutton_white.svg" /> -->
+    </a>
+  </div>
+</div>
+
```

Comment:
@EdenSG FYI I'm adding another download page which should look identical with dl-vibecheck except for 2 things
- text in the center
- download link on the button